cdc/kv/token_region_test.go

```diff
@@ -43,31 +41,33 @@ func (s *tokenRegionSuite) TestRouter(c *check.C) {
 	for i := 0; i < limit; i++ {
 		select {
 		case sri := <-r.Chan():
-			c.Assert(sri.ts, check.Equals, uint64(i))
+			require.Equal(t, uint64(i), sri.ts)
 			r.Acquire(store)
 			regions = append(regions, sri)
 		default:
-			c.Error("expect region info from router")
+			t.Error("expect region info from router")
 		}
 	}
-	c.Assert(r.tokens[store], check.Equals, limit)
+	require.Equal(t, limit, r.tokens[store])
 	for range regions {
 		r.Release(store)
 	}
-	c.Assert(r.tokens[store], check.Equals, 0)
+	require.Equal(t, 0, r.tokens[store])
 }
 
-func (s *tokenRegionSuite) TestRouterWithFastConsumer(c *check.C) {
-	defer testleak.AfterTest(c)()
-	s.testRouterWithConsumer(c, func() {})
+func TestRouterWithFastConsumer(t *testing.T) {
+	defer testleak.AfterTestT(t)
+	t.Parallel()
+	testRouterWithConsumer(t, func() {})
 }
 
-func (s *tokenRegionSuite) TestRouterWithSlowConsumer(c *check.C) {
-	defer testleak.AfterTest(c)()
-	s.testRouterWithConsumer(c, func() { time.Sleep(time.Millisecond * 15) })
+func TestRouterWithSlowConsumer(t *testing.T) {
+	defer testleak.AfterTestT(t)
```

Comment:
Why not remove this line: `defer testleak.AfterTestT(t)`?

Reply:
Already removed, PTAL